### PR TITLE
[newrelic-logging]: fix missing propagation of priorityClassName

### PR DESCRIPTION
**Pull request by woehrl01**
#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - priortyClassName can be defined via global, but is not propgated to the client, resulting in the value not beeing set.

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [ ] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[mychartname]`)

### CODE REVIEW

File: charts/newrelic-logging/templates/daemonset-windows.yaml [hunk 1]
```diff
@@ -43,6 +43,9 @@ spec:
       {{- if $.Values.image.pullSecrets }}
       imagePullSecrets:
 {{ toYaml $.Values.image.pullSecrets | indent 8 }}
+      {{- end }}
+      {{- with include "newrelic.common.priorityClassName" . }}
```
Comment (danybmx) [inline charts/newrelic-logging/templates/daemonset-windows.yaml:47]: Hi! This should be `{{- with include "newrelic.common.priorityClassName" $ }}` instead as for windows daemonset we're iterating over versions

**Review by danybmx — CHANGES_REQUESTED** [review]
Hi @woehrl01, thanks a lot for the contribution!
I've left a comment as I think this will fail on windows nodes.

File: charts/newrelic-logging/templates/daemonset-windows.yaml (hunk 1 above)
Comment (woehrl01) [reply charts/newrelic-logging/templates/daemonset-windows.yaml:47]: Thanks, I'll add the changes in the next days

Comment (woehrl01) [reply charts/newrelic-logging/templates/daemonset-windows.yaml]: @danybmx Just pushed a new version, I also saw that there is already a priorityClassName set, but not from "commons" so I removed that one on windows.

**Pull request closed without merging**